provider/azure_test.go

```diff
@@ -18,9 +18,11 @@ package provider
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
 	"testing"
 
-	"github.com/Azure/azure-sdk-for-go/arm/dns"
+	"github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2018-05-01/dns"
```

Comment:
is this the latest version which works?

Reply:
Yes:
https://github.com/Azure/azure-sdk-for-go/tree/master/services/dns/mgmt